README.md

```diff
@@ -28,15 +28,28 @@ like the Clang-as-a-library compiler implementation.
 
 libVersioningCompiler uses the cmake build system.
 If you want to install libVersioningCompiler
-
+install the dependencies with LLVM version `${LLVM_V}`:
+
+```bash
+wget https://apt.llvm.org/llvm.sh
+chmod +x llvm.sh
+[sudo] ./llvm.sh ${LLVM_V} all
+[sudo] apt-get install -y git cmake uuid-dev g++-10 libz-dev
+[optional][sudo] update-alternatives --install /usr/bin/clang++ clang++ /usr/bin/clang++-15 100
+[optional][sudo] update-alternatives --install /usr/bin/clang clang /usr/bin/clang-15 100
+[optional][sudo] update-alternatives --install /usr/bin/opt opt /usr/bin/opt-15 100
+```
 clone the repository to `${LIBVC_ROOT}`
 
 ```
 cd ${LIBVC_ROOT}
 mkdir build
 cd build
-cmake .. [-DCMAKE_INSTALL_PREFIX="/path/to/your/custom/install/folder/"]
+CXX=/bin/g++-10 cmake .. [-DCMAKE_INSTALL_PREFIX="/path/to/your/custom/install/folder/"]
```

Comment:
Not sure why you need to manually specify the compiler.

If you need so, I would prefer to move it as cmake variable rather than env variable. I.e. `-DCMAKE_CXX_COMPILER=...`

Reply:
Same as [here](https://github.com/skeru/libVersioningCompiler/pull/3?notification_referrer_id=NT_kwDOAh-ycbM0NTExNTQ2NTY4OjM1NjMxNzI5#discussion_r985247312)

Reply:
by default, cmake should try to use the command `cxx` to compile cpp files, and `cxx` should always point to a valid cpp compiler installed in the system. The readme should not suggest the user to redirect the compilation to any specific compiler unless the suggested one is the only compatible compiler due to intrinsics in the code (and I think this is not our case)